components/organisms/app/HeaderNavigation.vue

```diff
@@ -1,93 +1,55 @@
 <template>
-  <nav class="relative w-full h-full flex items-center justify-center">
+  <nav class="relative flex items-center justify-center w-full h-full">
     <Link
-      v-for="{ title, href, slug, blank } in links"
+      v-for="{ title, href, slug, blank, icon, to } in links"
       :key="slug"
       :aria-label="title"
-      class="
-        relative
-        capitalize
-        font-medium
-        flex flex-col
-        items-center
-        justify-center
-        text-center
-        px-4
-        h-full
-        flex
-        items-center
-      "
-      :to="href"
+      class="relative flex flex-col items-center justify-center h-full px-4 font-medium text-center capitalize group "
+      :to="href || to"
       :blank="blank"
       :class="{
         'text-primary': currentSlug === slug,
         'hover:d-primary-text-hover': currentSlug !== slug
       }"
     >
-      {{ $t(`header.${title}`) }}
+      <div class="flex items-center">
+          {{ title }}
+          <Component v-if="icon" :is="icon" class="w-4 h-4 ml-2 text-gray-400 opacity-0 group-hover:opacity-100"/>
```

Comment:
I guess a md:group-hover

Reply:
@Atinux would you mean only for desktop in this case i think is a lg:group-hover ?

Reply:
Yes